Server/Components/Pawn/Scripting/Object/Natives.cpp

```diff
@@ -282,6 +282,7 @@ SCRIPT_API(GetObjectMaterial, bool(IObject& object, int materialIndex, int& mode
 	bool result = object.getMaterialData(materialIndex, data);
 	if (result)
 	{
+		modelid = data->model;
 		textureLibrary = data->textOrTXD;
 		textureName = data->fontOrTexture;
 		materialColour = data->materialColour.RGBA();
```

Comment:
Not directly related to this PR but this should probably return ARGB to be consistent with `SetObjectMaterial`.

Reply:
That's also true for font and background colors in `GetObjectMaterialText`.